client/app/components/composites/AvatarDropdown/AvatarDropdown.js

```diff
@@ -38,18 +39,25 @@ class AvatarDropdown extends Component {
   render() {
     const touchClass = isTouch ? '' : css.touchless;
     const openClass = this.state.isOpen ? css.openDropdown : '';
+    const notificationsClass = this.props.notificationCount > 0 ? css.hasNotifications : null;
+    const notificationBadgeInArray = this.props.notificationCount > 0 ?
+      [r(NotificationBadge, { className: css.notificationBadge }, this.props.notificationCount)] :
+      [];
     return div({
       onClick: this.handleClick,
       onBlur: this.handleBlur,
       tabIndex: 0,
-      className: classNames(this.props.className, touchClass, openClass, css.avatarDropdown),
+      className: classNames(this.props.className, touchClass, openClass, css.avatarDropdown, notificationsClass),
     }, [
-      r(Avatar, this.props.avatar),
+      div({ className: css.avatarWithNotifications }, [
+        r(Avatar, this.props.avatar),
+      ].concat(notificationBadgeInArray)),
```

Comment:
Isn't spread for object literals. I'm conditionally adding stuff to child-component array.